### PR TITLE
chore: change loading screen background shader

### PR DESCRIPTION
**Pull request by olavra**
- Sorted synchronization between fade in and out of tips with background color change
- Added async function to change background material color
- Tunned down postprocessing bloom

## What does this PR change?

...

## How to test the changes?

1. Launch the explorer
2. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs [hunk 1]
```diff
@@ -124,5 +124,23 @@ public async UniTask HideTipWithFadeAsync(int index, float duration, Cancellatio
...
             tips[index].gameObject.SetActive(false);
         }
+
+        public async UniTask ChangeBackgroundColor(Material material, string property, Color toColor, float duration, CancellationToken cancellationToken = default)
+        {
+            Color currentColor = material.GetColor(property);
+            float time = 0f;
+            while (time < duration)
+            {
+                if (cancellationToken.IsCancellationRequested)
```
Comment (lorux0) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs:134]: No need to check if cancelled, just do: `cancellationToken.ThrowIfCancellationRequested();`

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs [hunk 2]
```diff
@@ -185,7 +185,8 @@ private void ShowTip(int index)
 
             viewInstance.HideAllTips();
             viewInstance.ShowTip(index);
-            viewInstance.Background.color = tips.Tips[index].BackgroundColor;
+
+            viewInstance.Background.material.SetColor("_Color1", tips.Tips[index].BackgroundColor);
```
Comment (lorux0) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs:189]: This should be a responsibility of the view. You can encapsulate the complexity into a method: `viewInstance.SetBackgroundColor(tips.Tips[index].BackgroundColor)`

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs [hunk 3]
```diff
@@ -202,16 +203,13 @@ async UniTaskVoid ShowTipWithFadeAsync(CancellationToken ct)
...
-                async UniTask HideAndShowAsync()
-                {
-                    await viewInstance.HideTipWithFadeAsync(prevTip, 0.5f, ct);
-                    await viewInstance.ShowTipWithFadeAsync(index, 0.5f, ct);
-                }
+                await viewInstance.HideTipWithFadeAsync(prevTip, 0.3f, ct);
 
-                await UniTask.WhenAll(backgroundTask, HideAndShowAsync());
+                await UniTask.WhenAll(
+                    viewInstance.ChangeBackgroundColor(viewInstance.Background.material, "_Color1", tips.Tips[index].BackgroundColor, 0.3f, ct),
```
Comment (lorux0) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs:210]: Feature envy smell. You should be able to do: `viewInstance.SetBackgroundColor(tips.Tips[index].BackgroundColor, 0.3f, ct)`

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs [hunk 4]
```diff
@@ -124,5 +124,23 @@ public async UniTask HideTipWithFadeAsync(int index, float duration, Cancellatio
 
             tips[index].gameObject.SetActive(false);
         }
+
+        public async UniTask ChangeBackgroundColor(Material material, string property, Color toColor, float duration, CancellationToken cancellationToken = default)
```
Comment (lorux0) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs:128]: This method should simplify its parameters:
```
public async UniTask SetBackgroundColor(Color toColor, float duration, CancellationToken cancellationToken = default)
```

**Review by lorux0 — CHANGES_REQUESTED** [review]

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs (hunk 3 above)
Comment (olavra) [reply Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs:210]: I want to wait until the other tip is hidden to change the background color and display the tip at the same time

Comment (lorux0) [reply Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs:210]: Its ok, you can make `viewInstance.ChangeBackgroundColor` function an async task

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs (hunk 1 above)
Comment (olavra) [reply Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs]: Changed

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs (hunk 2 above)
Comment (olavra) [reply Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs]: Changed

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs (hunk 4 above)
Comment (olavra) [reply Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs]: changed

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs [hunk 5]
```diff
@@ -124,5 +124,27 @@ public async UniTask HideTipWithFadeAsync(int index, float duration, Cancellatio
 
             tips[index].gameObject.SetActive(false);
         }
+
+        public async UniTask ChangeBackgroundColorFade(Color toColor, float duration, CancellationToken ct)
+        {
+            string property = "_Color1";
```
Comment (lorux0) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs:130]: Can be converted into const

**Review by lorux0 — APPROVED** [review]

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs (hunk 5 above)
Comment (olavra) [reply Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs]: Done

**Review by davidejensen — APPROVED** [review]

**Pull request merged**